src/styles/stars.css

```diff
@@ -0,0 +1,29 @@
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
```

Comment:
Move it to other css file, it does not apply to this block.

Reply:
+